### ISSUE TITLE
[UI] Unable to see long paths in "Output" section of the UI

### ISSUE THREAD

**Issue by Minibus93** [HACKERALERT/Picocrypt#78]
Hello, 

If output file is long enough you are not able to see it completely 
Maybe making that box horizontally scrollable could be a fix, but any solution would be fine

Below an highlighted graphical explaination of the issue:
![Immagine](https://user-images.githubusercontent.com/73554423/163347349-dc87458e-5676-4f2e-8354-b5ae565a778d.png)

**Comment by HACKERALERT** [comment]
One solution I've considered is to just show the filename instead of the entire path. So just "Immatricola uni.zip.pcv" in the box. What do you think?

**Comment by Minibus93** [comment]
> One solution I've considered is to just show the filename instead of the entire path. So just "Immatricolazione uni.zip.pcv" in the box. What do you think?

This would be 10/10.
It's probably the most intuitive approach since a not very savvy person could not even know what a path is

**Comment by HACKERALERT** [comment]
Sounds good! It's an easy change and it'll be in the next release. One more question, should the text be centered in the box or left-aligned?

**Comment by Minibus93** [comment]
> Sounds good! It's an easy change and it'll be in the next release. One more question, should the text be centered in the box or left-aligned?

Left aligned I guess? 
Everything else seems left aligned so it would be a sort of consistency

**Comment by HACKERALERT** [comment]
Makes sense. I'll fix this up and it should be reflected in v1.26, which I plan to release within 3 or 4 days.

**Comment by HACKERALERT** [comment]
Fixed!
![Screenshot (295)](https://user-images.githubusercontent.com/48808396/163468697-472c30ef-66ab-404b-9f30-0333ee3e3458.png)

**Comment by HACKERALERT** [comment]
I've release v1.26 with this change, so it's all fixed. I'll close this issue now, but feel free to reopen if necessary.
